src/http_crawler/__init__.py

```diff
@@ -19,7 +19,10 @@ def crawl(base_url, follow_external_links=True):
 
     while todo:
         url = todo.pop()
-        rsp = session.get(url)
+        rsp = session.get(url, allow_redirects=follow_redirects)
+
+        if rsp.status_code // 100 == 3:
+            continue
```

Comment:
Hi @rkrp.  Sorry for the slow reply -- have been AFK for a few days.

I understand your confusion, and I think it's because the wording of #5 is unclear.  I'm sorry about that!